gym_roboy/envs/ros_proxy.py

```diff
@@ -71,6 +70,7 @@ def _create_ros_client(self, process_idx: int):
         self.step_client = self.node.create_client(GymStep, '/instance' + str(process_idx) + '/gym_step')
         self.reset_client = self.node.create_client(GymReset, '/instance' + str(process_idx) + '/gym_reset')
         self.goal_client = self.node.create_client(GymGoal, '/instance' + str(process_idx) + '/gym_goal')
+        self.observation_client = self.node.create_client(GymStep, '/instance' + str(process_idx) + '/gym_observation')
```

Comment:
Can we rename the service to avoid confusion with the MsjEnv obs? What `about gym_read_state/` ?